src/pages/index.tsx

```diff
@@ -1,19 +1,126 @@
-import { Flex, Text } from "@chakra-ui/react";
+import {
+  Flex,
+  Text,
+  Box,
+  Heading,
+  Button,
+  HStack,
+  Divider,
+  Stack,
+  Link,
+} from "@chakra-ui/react";
 import Image from "next/image";
 import FirstLandingImage from "public/images/landing_first.svg";
 import SecondLandingImage from "public/images/landing_second.svg";
 import ThirdLandingImage from "public/images/landing_third.svg";
 
 function LandingPage() {
   return (
-    <Flex height="100%" justifyContent="space-around" alignItems="center">
-      <Text fontSize="2xl">Landing Page.</Text>
-      <Image
-        src={FirstLandingImage}
-        alt="Build Websites"
-        width={500}
-        height={500}
-      />
+    <Flex direction="column" h="calc(100vh - 100px)">
+      <Stack
+        justifyContent="center"
+        alignItems="center"
+        direction={{ base: "column-reverse", md: "row" }}
+        marginBottom="10"
+        flex="1"
+        flexBasis="250px"
+      >
+        <Box bg="white" w={{ md: "50%" }} p={10}>
```

Comment:
Consider using absolute units for the width here.  On desktop the image gets pushed too far to the right.